### ISSUE TITLE
Problem with dragging several cells at same time

### ISSUE THREAD

**Issue by HaidenTenno** [c-villain/SwipeActions#9]
Hi! Your lib works perfectly for my app, but i noticed strange behavior: 

I need to show button only for single cell per time, and SwipeState property works good, but if you swipe multiple cells at same time, then several cells remain open.

https://user-images.githubusercontent.com/22249849/212084215-cf3801b6-881a-4638-9bc5-5d03cb8902c8.mp4

Is it a bug, or feature?
Thank you!

**Comment by c-villain** [comment]
@HaidenTenno Petr, hi! Thx for sharing! That's interesting question! How do u think the behaviour should be?🤔

**Comment by HaidenTenno** [comment]
Something like telegram or whatsapp have in chat list (it's just not possible to swipe several items at same time 😅)

**Comment by c-villain** [comment]
@HaidenTenno sounds good! Actually you can [disable](https://stackoverflow.com/questions/67455293/how-to-disable-multi-touch-on-entire-app-or-just-a-view-using-swiftui) multitouch drag or you have to support multitouch, haven't you?...
All in all that's really good notice! I'll try to found some decision)

**Comment by c-villain** [comment]
@HaidenTenno I've just checked this [decision](https://stackoverflow.com/questions/67455293/how-to-disable-multi-touch-on-entire-app-or-just-a-view-using-swiftui). It works fine!

You should set 
```swift
UIView.appearance().isMultipleTouchEnabled = false
UIView.appearance().isExclusiveTouch = true
```

e.g. in `onAppear` for ur RootView

Check this out!

**Comment by c-villain** [comment]
@HaidenTenno yeah! Thx for sharing!🤝

**Comment by c-villain** [comment]
@HaidenTenno FYI

Due to SwiftUI philosophy is not quite possible control multi touching in general and dragging several cells particularly. Anyway we can disable multitouch using UIKit by adding exclusive touch flag ... For this one in the release 4.0.0 of this SPM you can add special view modifier: `.allowMultitouching(false)` to your view strictly before using swipe actions:

```swift
...
YourView(...)
      .allowMultitouching(false) // <= look here
      .addSwipeAction( ...) {
        ...
      }
...
```

But as soon as this solution is based on UIKit you can't optimise rendering of `YourView` with `drawingGroup()`:
```swift
...
YourView(...)
      .allowMultitouching(false) // <= look here
      .addSwipeAction( ...) {
        ...
      }
      .drawingGroup() // <=  ❌ DON'T DO THAT
...
```

You'll definitely get this:

<img width="338" alt="Screenshot 2024-09-09 at 15 22 35" src="https://github.com/user-attachments/assets/b8671c69-a5e1-4ca7-8d62-67b68bd2d33f">

Actually rendering will be optimizing by SwiftUI engine... For full control you can add view modifier `.identifier(your id)` to `YourView(...)`

**Comment by c-villain** [comment]
fix in [0.4.0](https://github.com/c-villain/SwipeActions/releases/tag/0.4.0)
